tools/applicationinsights-web-snippet/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/applicationinsights-web-snippet",
-    "version": "1.1.6",
+    "version": "1.1.0",
```

Comment:
The last released version of this package was 1.0.1, so rolling back this number to the next release